### ISSUE TITLE
[question] The Page factory function that existed in Appium dotnet-Client has been removed. Could we add PageFactory functionality as an extension?

### ISSUE THREAD

**Issue by nm-free** [Dreamescaper/DotNetSeleniumExtras#11]
The Page factory function that existed in Appium dotnet-Client has been removed.

https://github.com/appium/dotnet-client/pull/472/commits/6294052fbe38e1a4f6bfda61312326e397ed8e2#diff-a7defabfd952cf3ef940d64c43fbcf3bcf3bcf3bcf3bcf3bcf3bcf3bcf 6BCE6B6

Removed appium dotnet-client pagefacotory property from appium.

FindsByUIAutomatorsAttribute
FindsByAndroidUIAutomatorAttribute

Could we add PageFactory functionality as an extension?

**Comment by Dreamescaper** [comment]
I haven't really used Appium, but you should be able to create your attribute by inheriting from AbstractFindsByAttribute.
Something like that:

```csharp
    public class FindsByAndroidUIAutomatorAttribute(string selector) : AbstractFindsByAttribute
    {
        public override By Finder => new ByAndroidUIAutomator(selector);
    }
```

**Comment by namilkimfree** [comment]
https://github.com/appium/dotnet-client/pull/472/files


I am wondering if I can just use the decorator provided by Selenium and implement only the attribute class without implementing a separate decorator.

@Dreamescaper

**Comment by Dreamescaper** [comment]
Yes, you don't need to reimplement any decorators, it should work simply with that attribute.
